apis/v1alpha2/validation/httproute_test.go

```diff
@@ -868,7 +868,7 @@ func TestValidateHTTPRouteTypeMatchesField(t *testing.T) {
 				StatusCode: new(int),
 			},
 		},
-		errCount: 0,
+		errCount: 1,
```

Comment:
why has this changed ?

Reply:
The same reason that I mentioned for `v1beta1` changes.

Reply:
@slayer321 do you happen to have a link handy to that comment or can you just copy it here? Not finding the context easily and also have the same question.

Reply:
I was taking about this comment https://github.com/kubernetes-sigs/gateway-api/pull/2161#discussion_r1251477315